src/lib/generate-site-name.ts

```diff
@@ -23,22 +23,30 @@ export function generateSiteName( usedSiteNames: string[] ): string {
 		__( 'My Swift Website' ),
 		__( 'My True Website' ),
 	];
-	let proposedName = __( 'My WordPress Website' );
-	let tryCount = 0;
 
-	while ( usedSiteNames.includes( proposedName ) && tryCount < siteNames.length ) {
-		tryCount++;
-		proposedName = siteNames[ Math.floor( Math.random() * siteNames.length ) ];
+	const defaultName = __( 'My WordPress Website' );
+	if ( ! usedSiteNames.includes( defaultName ) ) {
+		return defaultName;
 	}
 
-	return proposedName;
+	const availableNames = siteNames.filter( ( name ) => ! usedSiteNames.includes( name ) );
+	if ( availableNames.length > 0 ) {
+		return availableNames[ Math.floor( Math.random() * availableNames.length ) ];
+	}
+
+	const randomName = siteNames[ Math.floor( Math.random() * siteNames.length ) ];
+	let postfix = 1;
```

Comment:
Should it be `suffix` instead of `postfix`? Or just a `siteNumber` to ensure it makes sense for LTR and RTL?

Reply:
> Should it be suffix instead of postfix? Or just a siteNumber to ensure it makes sense for LTR and RTL?

If we make it as suffix, then it will be confusing for LTR environments, since we used to see number at the end (e.g. in MacOS, Windows).
So my opinion, we should change postfix with suffix only for RTL.
WDYT?
<img width="206" alt="Screenshot 2025-02-10 at 11 49 22" src="https://github.com/user-attachments/assets/3bc0ddab-0c7d-43af-b83b-44d193bd245b" />

Reply:
My comment referred only to the variable name. `postfix` is the name of the mail-sending agent. I think it should be `affix`, instead, or even better, just `siteNumber`.